pkg/sync/v6/data/Role.rbac.authorization.k8s.io/openshift-infra/prometheus-k8s.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-infra
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - list
```

Comment:
https://github.com/openshift/cluster-monitoring-operator/blob/master/assets/prometheus-k8s/role-specific-namespaces.yaml
```
    - get
    - list
    - watch
```

Reply:
yes I see that's what is listed over in that file. However only `list` is necessary in practice ;)

Reply:
actually `list` and `get` are required afterall. Strange...I didn't see any complaints from the logs during my first tests last night... I've added a fix as a follow-up in https://github.com/openshift/openshift-azure/pull/1714/commits/b0cad95039775d6653c4b9b6c074f747590d5461